src/components/Activity/Connected.js

```diff
@@ -31,7 +31,7 @@ class Connected extends Component {
 
   onRequestCallback(result, error) {
     if (error) {
-      alert(JSON.stringify(error));
+      alert(error);
```

Comment:
Yeah... If bug free... no alert forever
and I'll try to find some way don't alert at release version